### PR TITLE
CYS: Remove strong tags from patterns

### PR DESCRIPTION
**Pull request by tjcafferkey**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Open the post editor.
2. Add `Banner` pattern.
3. Ensure that the header text isn't bold
4. Repeat 1-3 for `Hero Product 3 split pattern`, `Three Columns With Images And Content` and `Heading With Three columns Of Content With Link`

### Changelog entry

-   [x] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [x] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 
Remove strong tags from patterns

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

**Review by gigitux — APPROVED** [review]
LGTM! :shipit: 

@tjcafferkey can you take another look before merging it? Thanks!

File: plugins/woocommerce/patterns/heading-with-three-columns-of-content-with-link.php [hunk 1]
```diff
@@ -20,8 +20,8 @@
 <!-- /wp:spacer -->
 
 <!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignwide"><!-- wp:heading {"textAlign":"left","align":"full","style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
-<h2 class="wp-block-heading alignfull has-text-align-left" style="font-style:normal;font-weight:700"><?php echo esc_html( $header ); ?></h2>
+<div class="wp-block-group alignwide"><!-- wp:heading {"textAlign":"left","align":"full","style":{"typography":{"fontStyle":"normal"}}} -->
```
Comment (tjcafferkey) [inline plugins/woocommerce/patterns/heading-with-three-columns-of-content-with-link.php:23]: Doesn't this heading block need a `{"level":2}` attr?

File: plugins/woocommerce/patterns/three-columns-with-images-and-content.php [hunk 2]
```diff
@@ -23,8 +23,8 @@
 <!-- /wp:spacer -->
 
 <!-- wp:group {"align":"wide","layout":{"type":"constrained"}} -->
-<div class="wp-block-group alignwide"><!-- wp:heading {"textAlign":"left","align":"full","style":{"typography":{"fontStyle":"normal","fontWeight":"700"}}} -->
-<h2 class="wp-block-heading alignfull has-text-align-left" style="font-style:normal;font-weight:700"><strong><?php echo esc_html( $header ); ?></strong></h2>
+<div class="wp-block-group alignwide"><!-- wp:heading {"textAlign":"left","align":"full","style":{"typography":{"fontStyle":"normal"}}} -->
```
Comment (tjcafferkey) [inline plugins/woocommerce/patterns/three-columns-with-images-and-content.php:26]: Doesn't this heading block need a `{"level":2}` attr?

Comment (gigitux) [reply plugins/woocommerce/patterns/three-columns-with-images-and-content.php:26]: I don't think that it is necessary. Gutenberg parses correctly the block:


https://github.com/user-attachments/assets/955dd1cf-c75e-4e52-93a6-a2522d950b6d

**Review by tjcafferkey — COMMENTED** [review]
LGTM but I cant approve cos its my PR

**Pull request merged**